Formula/gnuradio.rb

```diff
@@ -90,8 +89,8 @@ def install
     ENV.prepend_create_path "PYTHONPATH", "#{venv_root}/lib/python#{xy}/site-packages"
     venv = virtualenv_create(venv_root, "python3")
 
-    %w[Mako Cheetah3 PyYAML click click-plugins].each do |r|
-      venv.pip_install resource(r)
+    resources.each do |r|
+      venv.pip_install r
     end
```

Comment:
I think this is does the same thing as
```suggestion
    venv.pip_install resources
```